iceoryx_hoofs/posix/filesystem/include/iox/detail/posix_acl.hpp

```diff
@@ -31,24 +32,22 @@
 
 namespace iox
```

Comment:
Since this file is in the `detail` folder, should we add the `detail` namespace here?

Reply:
Good point. There are already other classes in `detail` without the detail namespace. It seems we are a bit inconsistent here. Before the release we should look over all files in `detail` and decide what to do.

I think there might be use cases where one wants to move a header into `detail` and not have that namespace, e.g. if the user is supposed to include a specific header which includes the header from `detail` ... on the other hand, the common header could just add a using declaration to re-export everything the user needs in the iox namespace.

Nevertheless, I'll add the `detail` namespace for `PosixAcl`.

Reply:
Done. And sorry, I accidentally rebased to master